content/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site.md

```diff
@@ -105,6 +105,8 @@ To set up an apex domain, such as `example.com`, you must configure a custom dom
 
 {% data reusables.pages.www-and-apex-domain-recommendation %} For more information, see "[Configuring a subdomain](#configuring-a-subdomain)."
 
+1. Navigate to your DNS provider and create a `CNAME` record that points the `www` subdomain to the default domain for your site. For example, if you want to use the subdomain `www.example.com` for your user site, create a `CNAME` record that points `www.example.com` to `<user>.github.io`. If you want to use the subdomain `www.example.com` for your organization site, create a `CNAME` record that points `www.example.com` to `<organization>.github.io`. The `CNAME` record should always point to `<user>.github.io` or `<organization>.github.io`, excluding the repository name. {% data reusables.pages.contact-dns-provider %} {% data reusables.pages.default-domain-information %}
```

Comment:
```suggestion
Navigate to your DNS provider and create a `CNAME` record for the `www` subdomain that points to your {% data variables.product.prodname_pages %} default domain. For example, if your site is located at `<user>.github.io`, you should create a `CNAME` record that points `www.example.com` to `<user>.github.io` Similarly, for an organization site located at `<organization>.github.io`, you should create a `CNAME` record that points `www.example.com` to `<organization>.github.io`. Ensure that the `CNAME` record points directly to `<user>.github.io` or `<organization>.github.io` without including the repository name.

{% data reusables.pages.contact-dns-provider %} {% data reusables.pages.default-domain-information %}
```

Slightly modified for better readability. ⚡  Also, it doesn't need to be a numbered list.

Reply:
FYI, I didn't write that paragraph down, I just copied and only slightly modified it from here:

https://github.com/revolter/docs-1/blob/1801b6f218356fda2195a3b43c26f511adcd871a/content/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site.md?plain=1#L125